portal/src/graphql/portal/ApplicationsConfigurationScreen.module.scss

```diff
@@ -55,3 +55,19 @@
   pointer-events: none;
   opacity: 0.5;
 }
+
+.callout {
+  padding: 8px;
```

Comment:
Use `styles` to custom the component itself. Use `className` to layout the component within its parent.